src/main/kotlin/net/atos/zac/app/zaak/converter/RestZaakOverzichtConverter.kt

```diff
@@ -73,7 +73,7 @@ class RestZaakOverzichtConverter @Inject constructor(
         return RestZaakOverzicht(
             identificatie = zaak.identificatie,
             startdatum = takeIf { zaakrechten.lezen }?.let { zaak.startdatum },
-            toelichting = takeIf { zaakrechten.lezen }?.let { zaak.toelichting },
+            omschrijving = takeIf { zaakrechten.lezen }?.let { zaak.omschrijving },
```

Comment:
`takeIf` repeated a few times, seems very inefficient. The only thing needed is `zaakrechten.lezen`, so determine that once and set all 3 fields, ie:

```
        return RestZaakOverzicht(
            identificatie = zaak.identificatie
        ).apply {
            if (zaakrechten.lezen) {
                startdatum = zaak.startdatum,
                toelichting = zaak.toelichting,
                zaaktype = zaaktype.omschrijving,
            }
        }
```

Reply:
Applied